Makefile

```diff
@@ -1072,18 +1070,39 @@ libraryopt:
 partialclean::
 	$(MAKE) -C stdlib clean
 
-# The lexer and parser generators
+# The lexer generators
+
+ocamllex_MODULES = $(addprefix lex/,\
+  cset syntax parser lexer table lexgen compact  common output outputbis main)
+
```

Comment:
At this point, the additional warnings disabled by the merge could be re-enabled with:

```make
lex/%.cmo: OC_COMMON_CFLAGS+=-w +40+41+66
lex/%.cmi: OC_COMMON_CFLAGS+=-w +40+41+66
lex/%.cmx: OC_COMMON_CFLAGS+=-w +40+41+66
```